modules/navier_stokes/doc/content/source/fvbcs/INSFVSwitchableOutletPressureBC.md

```diff
@@ -0,0 +1,22 @@
+# INSFVSwitchableOutletPressureBC
+
+!syntax description /FVBCs/INSFVSwitchableOutletPressureBC
+
+The file is similar to [INSFVOutletPressureBC.md] but it allows the boundary conditions to be switched on/off.
+
+The switch works as follows:
+
+- If `switch = true`: the boundary condition is applied as described in [INSFVOutletPressureBC.md].
+
+- If `switch = false`: the boundary condition is not applied and a single sided extrapolation to the boundary
+  is applied from internal extrapolation. The user can expect second order convergence.
```

Comment:
can the user expect second order convergence since we use single-term boundary expansion?

Reply:
Agree. Removed the expectation for second order convergence in the text